compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/types/BLangTupleTypeNode.java

```diff
@@ -40,6 +40,8 @@ public class BLangTupleTypeNode extends BLangType implements TupleTypeNode {
     public List<BLangType> memberTypeNodes = new ArrayList<>();
     public BLangType restParamType;
 
+    public boolean isAnonymous;
```

Comment:
Why do we need a separate field for this? We can check via flags when necessary, right?

Reply:
Fixed.